overrides/main.html

```diff
@@ -5,92 +5,113 @@
 {% endblock %}
 
 {% block scripts %}
-    {{super()}}
-    
-    <script>
-    if (! navigator.share) { // API not working, then load the script
-      let script = document.createElement("script");
-      script.defer = true;
-      script.src = "{{ 'javascripts/sweetalert2@11.js' | url }}"; // https://cdn.jsdelivr.net/npm/sweetalert2@11
-      document.head.append(script);
-    }
-    function selectText(nodeId) {
-        const node = document.getElementById(nodeId);
+{{super()}}
 
-        if (document.body.createTextRange) {
-            const range = document.body.createTextRange();
-            range.moveToElementText(node);
-            range.select();
-        } else if (window.getSelection) {
-            const selection = window.getSelection();
-            const range = document.createRange();
-            range.selectNodeContents(node);
-            selection.removeAllRanges();
-            selection.addRange(range);
-        } else {
-            console.warn("Could not select text in node: Unsupported browser.");
-        }
-    }
+<script>
 
-    let btn = document.createElement("button");
-    btn.innerHTML = '{% include ".icons/material/share-variant.svg" %}';
-    btn.classList.add("md-header__button", "md-icon","twemoji");
-    var title = document.getElementsByClassName("md-header__inner")[0];
-    title.insertBefore(btn, document.getElementsByClassName("md-header__option")[0]);
+	function load_style(href) {
+		let stylesheet = document.createElement("link");
+		stylesheet.rel = "stylesheet";
+		stylesheet.href = href;
+		document.head.append(stylesheet);
+	}
 
-    btn.addEventListener('click', event => {
-        var title = document.title;
-        var url = window.location.href;
+	function load_script(src) {
+		let script = document.createElement("script");
+		script.defer = true;
+		script.src = src;
```

Comment:
```suggestion
		script.src = src;
		script.async = false;
```
If you notice, this error is happening irregularly, one fix would be to explicitly set aync to false hence all the scripts will load in order and not in a threaded fashion. This issue is caused when custom.js is loaded before katex.js